test/eventinge2erekt/features/encryption.go

```diff
@@ -0,0 +1,202 @@
+package features
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/url"
+	"strings"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+	"knative.dev/pkg/injection/clients/dynamicclient"
+	"knative.dev/pkg/logging"
+	"knative.dev/reconciler-test/pkg/environment"
+	"knative.dev/reconciler-test/pkg/feature"
+	"knative.dev/reconciler-test/pkg/resources/knativeservice"
+)
+
+func VerifyEncryptedTrafficToActivatorToApp(refs []corev1.ObjectReference, since time.Time) *feature.Feature {
+	// Just a small sleep to let the logs be written.
+	time.Sleep(10 * time.Second)
```

Comment:
why the sleep instead of the usual "wait for xyz to happen before failing" ?

Reply:
I can probably get rid of it. I took this from our downstream tests but now in this PR I throw an error when no log lines match: 
```
if encrypted == 0 && unencrypted == 0 {
    return fmt.Errorf("no log lines matching filter")
}
```
So, it should be possible to wait for no errors.